openupgrade_scripts/scripts/website/14.0.1.0/end-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+from openupgradelib import openupgrade, openupgrade_140
+
+
+def convert_field_html_string(env):
+    fields = env["ir.model.fields"].search(
+        [("ttype", "=", "html"), ("store", "=", True)]
+    )
+    for field in fields:
+        model = field.model_id.model
+        if env.get(model):
```

Comment:
Sorry, I told you this incorrectly. It should be:

```suggestion
        if env.get(model, False) is not False:
```

Reply:
done!